ceph-ansible.spec.in

```diff
@@ -35,6 +35,14 @@ for f in ansible.cfg *.yml *.sample group_vars roles library plugins infrastruct
   cp -a $f %{buildroot}%{_datarootdir}/ceph-ansible
 done
 
+# Strip coreos files.
+# These are unneeded on RPM-based distros, and the playbooks download random
+# things from around the internet.
+pushd %{buildroot}%{_datarootdir}/ceph-ansible
+  rm -r roles/ceph-common-coreos
```

Comment:
On some distros 'rm' asks you to confirm, `-f` will force the deletion. If you're sure you down have that problem then we can go with this.